src/ontology/mondo-edit.obo

```diff
@@ -1922,16 +1922,16 @@ subset: gard_rare {source="MONDO:GARD"}
 subset: otar {source="MONDO:OTAR"}
 subset: rare
 synonym: "GLUT1 deficiency syndrome" EXACT CLINGEN_LABEL []
```

Comment:
can we add a gene association for this term?

Reply:
this has the axiom:
'disease has basis in disruption of' some 'D-glucose transmembrane transporter activity'

I don't think we normally add associated genes for gene deficiency syndromes, it seems like we add axioms that describe the disruption of the biological process